its/plugin/pom.xml

```diff
@@ -9,9 +9,9 @@
     <version>1.6.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>it-xml</artifactId>
+  <artifactId>it-xml-plugin</artifactId>
 
-  <name>SonarXML :: IT</name>
+  <name>SonarXML :: IT Plugin</name>
```

Comment:
remove orchestator version in this file

Reply:
Let's do that in another PR, as I would also like to update version of SQ WebService API which is used